server/src-lib/Hasura/GraphQL/Execute/LiveQuery/Poll.hs

```diff
@@ -180,69 +210,65 @@ pushResultToCohort
   -> Maybe ResponseHash
   -> LiveQueryMetadata
   -> CohortSnapshot
-  -> IO ()
+  -> IO ([SubscriberId], [SubscriberId])
+  -- ^ (subscribers to which data has been pushed, subscribers which already
+  -- have this data)
```

Comment:
I’d appreciate an explicit comment that notes this information is only used for reporting:
```suggestion
  -- ^ subscribers to which data has been pushed, subscribers which already
  -- have this data (this information is exposed by metrics reporting)
```